packages/paste-core/primitives/text/__tests__/StyleFunctions.spec.tsx

```diff
@@ -1,194 +1,196 @@
 import {getPseudoStyles, getCustomElementStyles} from '../src/StyleFunctions';
 
-// When rendered the theme object is injected onto Box props via emotions theming context
-// which is where the css and variant style functions look for their values
-const mockTheme = {
-  theme: {
-    space: {
-      space10: '10px',
-    },
-    backgroundColors: {
-      colorBackgroundPrimary: 'blue',
-    },
-    borderColors: {
-      colorBorderError: 'red',
-    },
-    textColors: {
-      colorTextSuccess: 'green',
-    },
-    elements: {
-      CARD: {
-        color: 'colorTextSuccess',
-        borderColor: 'purple',
-        ':hover': {
-          cursor: 'pointer',
-        },
-        variants: {
-          primary: {
-            backgroundColor: 'colorBackgroundPrimary',
-            ':hover': {
-              margin: 'space10',
-              textDecoration: 'underline',
+describe('Text StyleFunctions', () => {
+  // When rendered the theme object is injected onto Box props via emotions theming context
+  // which is where the css and variant style functions look for their values
+  const mockTheme = {
+    theme: {
+      space: {
+        space10: '10px',
+      },
+      backgroundColors: {
+        colorBackgroundPrimary: 'blue',
+      },
+      borderColors: {
+        colorBorderError: 'red',
+      },
+      textColors: {
+        colorTextSuccess: 'green',
+      },
+      elements: {
+        CARD: {
+          color: 'colorTextSuccess',
+          borderColor: 'purple',
+          ':hover': {
+            cursor: 'pointer',
+          },
+          variants: {
+            primary: {
+              backgroundColor: 'colorBackgroundPrimary',
+              ':hover': {
+                margin: 'space10',
+                textDecoration: 'underline',
+              },
             },
           },
         },
-      },
-      ALERT: {
-        padding: 'space10',
-        textDecoration: 'underline',
-        variants: {
-          foo: {
-            borderColor: 'colorBorderError',
+        ALERT: {
+          padding: 'space10',
+          textDecoration: 'underline',
+          variants: {
+            foo: {
+              borderColor: 'colorBorderError',
+            },
           },
         },
       },
     },
-  },
-};
+  };
```

Comment:
This is a little memory leak. I'm pretty sure that because mockTheme is outside the describe block and is a global, for what ever reason it's not torn down after the test, so it keeps a reference of it in memory. 

Regardless if you run `yarn jest --detectLeaks` before and after, this test stops being flagged as a memory leak.